src/clients/firstboot_hostname.rb

```diff
@@ -94,9 +107,8 @@ def HostnameDialog
         Frame(
           _("Hostname and Domain Name"),
           VBox(
-            HBox("HOSTNAME", HSpacing(1), "DOMAIN"),
-            Left("DHCP_HOSTNAME"),
-            Left("WRITE_HOSTNAME")
+            Left("HOSTNAME"),
+            Left("DHCP_HOSTNAME")
```

Comment:
Just curiosity, why is this change needed?

Reply:
Because the hostname settings have been simplified. Now there are only two fields: _hostname_ and _set hostname via DHCP_. The _domain_ and _write hostname_ settings have been dropped.